atlp-rwanda/hackers-ec-Fe
# #187419032 A buyer should be able to track their order status

### What does this PR do?
A buyer should be able to track their order status.

### How should this be manually tested?
1. Clone the repo and cd into it
2. Checkout to ```ft-buyer-tracking-order-187419032``` and ```run npm install```
3. Run npm run dev to start the application locally
4.  Click   ```My Orders``` in Navbar and  see all order information, even single order.

### Pivot tracker Id
#187419032
